docs/modules-03-theming.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: Theming
+permalink: /modules/theming/
+resource: true
+categories: Modules
+---
+{% include base.html %}
+
+The following CSS classes allow you to control theming.
+
+### Colours
+Use predefined colors agnostically.
+
+- **Responsive:** No
+
+{% highlight scss %}
+.color--light {
+  color: $white;
+}
+
+.color--dark {
+  color: $text-color;
+}
+
+.color--primary {
+  color: darken($primary, 12%);
+}
+{% endhighlight %}
+
+
+### Background Colours
+Use predefined colours and variant tints of `grey`, `blue` and `black`.
+
+- **Responsive:** No
+
+{% highlight scss %}
+// Black Tints
+
+{% endhighlight %}
+
+### Borders
+
+- **Responsive:** No
+
+{% highlight scss %}
+// Borders
+
+{% endhighlight %}
+
+### Box Shadow
+Predefined box-shadow values. Incrementally increasing with intensity.
```

Comment:
> _Incrementally increasing with intensity._

The above isn't necessarily accurate, as these presets do vary widely. Let's remove this sentence until we decide how to normalise them at a later time.